### DISCUSSION TITLE
Dark theme for the app?

### DISCUSSION THREAD

**Discussion by Proutbedaine** [hsoft/dupeguru#965] [Ideas]
I think there's not enough app that offer this now that we're in 2021, but a dark theme for any app should be a standard no? I mean, I do admit I have no idea how hard it can possibly be to do such a thing. Let's say, make it look like GitHub for example! I think this is one of the best example of color pallet for a dark theme and how they use it too; but doing something like that for these kinds of app, is it possible? And if so how hard is it?

Because personally I would highly appreciate it to be a thing. I really like to use this app full screen. The thing is though, I'm on a QLED 55" that can get REALLY bright if everything is white. With dark themes and any other stuff (movies, YouTube, gaming) the level of brightness is perfect and make every color pop. But then, when I fall on something I really wanna use fullscreen or even half-screen that is overly white, it burns both mine and my girlfriend's retinas hahaha. So yeah it's not the best. Especially since I tend to be a night owl so I'm in the dark and it gets pretty intense at times how the backlash from going to everything very dark to a super bright white screen all of a sudden hahaha.

So yeah if any of you would know if it'd be a possibility and also if it's something hard to do in general it'd be amazing please! <3

**Comment by glubsy** [comment]
It's been requested in https://github.com/arsenetar/dupeguru/issues/900 already. If you use Linux, you can easily change the theme to be darker. No idea how to do this on Windows.

**Reply by Proutbedaine** [reply]
> It's been requested in #900 already. If you use Linux, you can easily change the theme to be darker. No idea how to do this on Windows.

Oh oki, my bad. And yeah I'm on Win10. Is it usually something hard to program in these kinds of apps. Like, would it be very time consuming to achieve in a release build and maintain it or? :o

**Comment by arsenetar** [comment]
I think a dark mode is something that can be done, although this is already supported via the OS in Mac OS and most Linux distributions (by matching the OS theme).  So this is mainly a windows specific item, unless we are looking to override the OS theme.

**Reply by Proutbedaine** [reply]
> I think a dark mode is something that can be done, although this is already supported via the OS in Mac OS and most Linux distributions (by matching the OS theme). So this is mainly a windows specific item, unless we are looking to override the OS theme.

Oh, yeah I'm on windows haha. And great! I hope it's something that'll become a reality eventually ^^.

**Reply by arsenetar** [reply]
Just FYI for anyone, there is now a Dark Theme option in preferences on Windows with the release of 4.2.1.  In other environments the app should follow the OS theme.
